internal/api/README.md

```diff
@@ -0,0 +1,39 @@
+# Commit0 Api
+
+## Usage
+ - To run:
+ `go run internal/api/generate_api.go internal/api/create_project.go`
```

Comment:
Let's change this to an argument to the binary. `commit0 walkthrough` or `commit0 ui` or something

Reply:
how do I do this? @bmonkman --ty!

Reply:
Yeah you would create a new file like `cmd/ui.go` which adds a new command